api/src/pcapi/repository/clean_database.py

```diff
@@ -86,6 +86,8 @@ def clean_all_database(*args, **kwargs):  # type: ignore [no-untyped-def]
     fraud_models.BeneficiaryFraudReview.query.delete()
     users_models.Token.query.delete()
     offers_models.OfferValidationConfig.query.delete()
+    offers_models.OfferValidationRule.query.delete()
+    offers_models.OfferValidationSubRule.query.delete()
```

Comment:
Puisque les "sub-rules" dépendent des "rules", ne faudrait-il pas plutôt supprimer dans l'ordre inverse ?

```suggestion
    offers_models.OfferValidationSubRule.query.delete()
    offers_models.OfferValidationRule.query.delete()
```